Unix/pal/palcommon.h

```diff
@@ -22,7 +22,7 @@
 #    include <windows.h>
 #    include <strsafe.h>
 #else
-#    include "../../common/linux/sal.h"
+#    include "linux/sal.h"
```

Comment:
yeah, I know that, but I just like things being relative to the top level. I am ok with Linux.sal.h but with angle-brackets which is used for doing search where quotes mean relative to current directory in general